cmd/modern/root/config.go

```diff
@@ -18,13 +20,14 @@ type Config struct {
 // DefineCommand defines the `sqlcmd config` command, which is only
 // more sub-commands (`sqlcmd config` does not `run` anything itself)
 func (c *Config) DefineCommand(...cmdparser.CommandOptions) {
+	const configCommand = "sqlcmd config use-context mssql"
 	options := cmdparser.CommandOptions{
 		Use:         "config",
-		Short:       `Modify sqlconfig files using subcommands like "sqlcmd config use-context mssql"`,
+		Short:       localizer.Sprintf(`Modify sqlconfig files using subcommands like "%s"`, configCommand),
 		SubCommands: c.SubCommands(),
 		Examples: []cmdparser.ExampleOptions{
 			{
-				Description: "Add context for existing endpoint and user (use SQLCMD_PASSWORD or SQLCMDPASSWORD)",
+				Description: localizer.Sprintf("Add context for existing endpoint and user (use SQLCMD_PASSWORD or SQLCMDPASSWORD)"),
```

Comment:
Instead of creating const variables to pass to Sprintf(), is it okay to use string literals directly as argument to Sprintf()?
`localizer.Sprintf("Add context for existing endpoint and user (use %s or %s)", "SQLCMD_PASSWORD", "SQLCMDPASSWORD"),
`

Reply:
sure. I equate literal and constant for this purpose.
That said, we could move the existing constant declarations of sqlcmd variable names from pkg\sqlcmd to something under internal in a future PR to improve reuse